### PR TITLE
[Fix](Lateral View) Error view includes lateral view

### LINKED ISSUE #9529: [Bug](Lateral View) Create incorrect view when view includes the lateral view
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

Doris 1.0.0

### What's Wrong?

ERROR 1054 (42S22): errCode = 2, detailMessage = Unknown column 'topic_id' in 't'

### What You Expected?

Return result

### How to Reproduce?

1. create table dwd_content_create_base with column topic_ids
2. create view
```
 CREATE VIEW `tmp_1` AS SELECT `t`.`topic_id` AS `topic_id` FROM (SELECT `topic_ids` AS `topic_ids` FROM `dwd_content_create_base`) t1 lateral view explode_split(`t1`.`topic_ids`, ',') t as topic_id;
```
3. query
```
select topic_id from tmp_1
```

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EmmyMiao87**
# Proposed changes

Issue Number: close #9529

## Problem Summary:

Fixed #9529

When the lateral view based on a inline view which belongs to a view,
Doris could not resolve the column of lateral view in query.
When a query uses a view, it mainly refers to the string representation of the view.
That is, if the view's string representation is wrong, the view is wrong.
The string representation of the inline view lacks the handling of the lateral view.
This leads to query errors when using such views.
This PR mainly fixes the string representation of inline views.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by HappenLee — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/TableRef.java [hunk 1]
```diff
@@ -627,20 +627,12 @@ public List<TupleId> getAllTableRefIds() {
     /**
      * Return the table ref presentation to be used in the toSql string
      */
+    // tbl1
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/TableRef.java:630]: write in java doc is better way

**Comment by morrySnow** [comment]
fe CheckStyle error need to be fixed

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**